templates/partials/_snap-details-video-layout.html

```diff
@@ -17,3 +17,18 @@
     </div>
   {% endif %}
 </div>
+
+<script>
+  document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Do you think this script would be better in the `_video.html` particle as its closer to the elements it's interacting with?

Reply:
Moved